build/webpack.base.config.js

```diff
@@ -54,6 +54,11 @@ function getCssLoaders(config) {
     return ['vue-style-loader', ...cssLoaders];
 }
 
+function fileLoaderName() {
+    const hash = process.env.NODE_ENV === 'production' ? '-[contenthash]' : '';
+    return `images/[name]${hash}.[ext]`;
```

Comment:
This provides better naming schemes for webpack-imported SVG files that do not get inlined or base-64 encoded